ios/Classes/FlutterBleLibPlugin.m

```diff
@@ -315,7 +315,7 @@ - (void)writeCharacteristicForService:(FlutterMethodCall *)call result:(FlutterR
 
 - (void)writeCharacteristic:(FlutterMethodCall *)call result:(FlutterResult)result {
     [_manager writeCharacteristic:[call.arguments[ARGUMENT_KEY_CHARACTERISTIC_IDENTIFIER] doubleValue]
-                      valueBase64:[self base64encodedStringFromBytes:call.arguments[ARGUMENT_KEY_BYTES]]
+                      valueBase64:[self base64encodedStringFromBytes:call.arguments[ARGUMENT_KEY_VALUE]]
```

Comment:
Why simulated writing to characteristics touches non-simulated iOS bridging library?

Reply:
I decided to change this name while implementing this feature instead of providing inconsistent naming conventions (in one place characteristic value param would be called bytes and in another place it would be called value). Moreover I didn't want to continue using "bytes" name because, as we already discussed, its wrong.

Reply:
I created a separate PR for FlutterBleLib https://github.com/Polidea/FlutterBleLib/pull/232